<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasvurularSayfasi</class>
 <widget class="QMainWindow" name="BasvurularSayfasi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>783</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>811</width>
      <height>711</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>10</y>
       <width>821</width>
       <height>691</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>-10</y>
        <width>450</width>
        <height>47</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="aramaKutusu">
         <property name="toolTip">
          <string>Aramak istediğiniz kelimeyi yazınız!</string>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(0, 170, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="araButonu">
         <property name="styleSheet">
          <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Ara</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tercihlerButonu">
         <property name="styleSheet">
          <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Tercihler</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>450</width>
        <height>46</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="tumBasvurularButonu">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>1</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1200</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left; /* Metni sola hizala */</string>
         </property>
         <property name="text">
          <string>   Tüm Başvurular</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="oncekiVitKontrolButonu">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>1</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1200</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left; /* Metni sola hizala */</string>
         </property>
         <property name="text">
          <string>Oncekı Vit Kontrolu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mukerrerKayitButonu">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>1</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1200</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left; /* Metni sola hizala */</string>
         </property>
         <property name="text">
          <string>Mukerrer Kayıt</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>450</width>
        <height>46</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="mentorGorusmesiTanimlananlarButonu">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>1</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left; /* Metni sola hizala */</string>
         </property>
         <property name="text">
          <string>   Mentor Görüşmesi Tanımlananlar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="basvuruFiltreliButonu">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>1</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1200</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left; /* Metni sola hizala */</string>
         </property>
         <property name="text">
          <string>Basvuru Fıltrelı</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>450</width>
        <height>46</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="mentorGorusmesiTanimlanmayanlarButonu">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>1</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left; /* Metni sola hizala */</string>
         </property>
         <property name="text">
          <string>   Mentor Görüşmesi Tanımlanmayanlanlar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="farkliKayitButonu">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>1</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1200</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left; /* Metni sola hizala */</string>
         </property>
         <property name="text">
          <string>Farklı Kayıt</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="Geri_Tusu">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>600</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="toolTip">
       <string>Tercihler Ekranina Geri Dön!</string>
      </property>
      <property name="statusTip">
       <string extracomment="Önceki sayfaya dön"/>
      </property>
      <property name="whatsThis">
       <string extracomment="Önceki sayfaya dön"/>
      </property>
      <property name="accessibleName">
       <string notr="true" extracomment="Önceki sayfaya dön"/>
      </property>
      <property name="accessibleDescription">
       <string extracomment="Önceki sayfaya dön"/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255, 0);
color: rgb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../geri_tusu.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Kapat_butonu_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>640</y>
        <width>45</width>
        <height>45</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>Uygulamadan cikar!</string>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
background: none;
border: none;
background-repeat: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>kapat.png</normaloff>kapat.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>811</width>
        <height>341</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Tarih</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ad Soyad</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mail</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Telefon</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Posta Kodu</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Eyalet</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Su Anki Durum</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ekonomik Durum</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="liste_kisi_sayisi">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>250</y>
        <width>401</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>50</y>
        <width>171</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>logo1.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>BASVURULAR</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>620</y>
        <width>811</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="geriButonu">
         <property name="styleSheet">
          <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>GERI</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cikisButonu">
         <property name="styleSheet">
          <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>EXIT</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>719</y>
      <width>861</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>©VIT | FHM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
